Readme.md

```diff
@@ -207,6 +207,11 @@ The options accepted have keys corresponding to the options described in [CLI Ar
   ignore: 'pid,hostname', // --ignore
   hideObject: false, // --hideObject
   singleLine: false, // --singleLine
+
+  dest: 1, // The file or file descriptor to write to
```

Comment:
what does default destination of 1 mean?

Reply:
the file descriptor of stdout.